subsys/osif/zb_nrf_platform.c

```diff
@@ -357,6 +370,14 @@ static void zboss_thread(void *arg1, void *arg2, void *arg3)
 	while (1) {
 		zboss_main_loop_iteration();
 	}
+
+#else
+	/* CONFIG_ZIGBEE_LIBRARY_NCP_DEV */
```

Comment:
Move to previous line together with `#else`?

Reply:
fixed